packages/chaire-lib-backend/src/config/server.config.ts

```diff
@@ -6,9 +6,78 @@
  */
 import path from 'path';
 import os from 'os';
+import _merge from 'lodash/merge';
 
-import config, { setProjectConfiguration } from 'chaire-lib-common/lib/config/shared/project.config';
+import config, {
+    ProjectConfiguration,
+    setProjectConfiguration as setProjectConfigurationCommon
+} from 'chaire-lib-common/lib/config/shared/project.config';
 import fs from 'fs';
+import { RoutingMode } from 'chaire-lib-common/lib/config/routingModes';
+
+export type TrRoutingConfig = {
+    /**
+     * Port to use for this trRouting instance
+     */
+    port: number;
+    /**
+     * If set to `true`, enable caching of connections for all scenarios in
+     * trRouting. Will use more memory
+     */
+    cacheAllScenarios: boolean;
+    // FIXME Do we need to configure a host here? If so, we need to properly
+    // support it in both batch and single calculation
+};
+
+// TODO Some project config option depend on the application, so should not be
```

Comment:
So the TODO means the routing section should be moved to the transition project and not have it here right ?  
Can we make sure we have an issue ? (Maybe add exemple of section in the comment)

Reply:
This TODO is copy-pasted from the chaire-lib-common's config file.

In the case we're considering, all routing engines are in chaire-lib already, so their configs need to be here too, otherwise TrRoutingProcessManager cannot access them if they are in Transition.